platformio.ini

```diff
@@ -19,7 +19,12 @@ libdeps_dir = .piolibdeps
 env_default = megaatmega2560
 
 [common]
-lib_deps = U8glib@1.19.1
+lib_deps =
+  U8glib@1.19.1
+  https://github.com/lincomatic/LiquidTWI2.git
+  https://github.com/teemuatlut/TMC2130Stepper.git
```

Comment:
Please use the library name and not git links. In the TMC SPI discussion it was discovered that PIO doesn't play nice with direct links.

Reply:
This approach seemed to be working without issue and is the same as in bugfix-2.0.x. I'd suggest we change it there if we're going to change it here.

Reply:
Yea the problem was with the LPC HAL and didn't link properly. It was fine for Teensy and AVR (possibly) so you wouldn't see a problem v1.1.x. It needs to be changed with v2.0 but I haven't bothered with it yet and it might just get included in some other PR.

Reply:
The LiquidTWI2 and TMC26XStepper libraries don't appear to be in the PlatformIO Library Manager, so I had to keep the git links for those. I was able to switch to using the library names for the others.